libnymea-zigbee/zigbeenode.cpp

```diff
@@ -266,20 +268,19 @@ ZigbeeReply *ZigbeeNode::readBindingTableEntries()
 
 void ZigbeeNode::initNodeDescriptor()
 {
-    qCDebug(dcZigbeeNode()) << "Request node descriptor from" << this;
+    qCDebug(dcZigbeeNode()) << "Requesting node descriptor from" << this;
     ZigbeeDeviceObjectReply *reply = deviceObject()->requestNodeDescriptor();
     connect(reply, &ZigbeeDeviceObjectReply::finished, this, [this, reply](){
         if (reply->error() != ZigbeeDeviceObjectReply::ErrorNoError) {
             qCWarning(dcZigbeeNode()) << "Error occured during initialization of" << this << "Failed to read node descriptor" << reply->error();
             m_requestRetry++;
```

Comment:
I think this is messing up the retry mechanism `m_requestRetry`. If you use a separate variable for each request type I think we should be fine doing it parallel.